pyaedt/maxwell.py

```diff
@@ -1045,3 +1045,52 @@ def assign_vector_potential(self, input_edge, vectorvalue=0, bound_name=None):
             self.boundaries.append(bound)
             return bound
         return False
+
+    @aedt_exception_handler
+    def assign_master_slave(self, master_edge, slave_edge, reverse_master=False, reverse_slave=False,
+                            same_as_master=True, bound_name=None):
+        """Assign Master/Slave Boundary Conditions to 2 edges of the same object.
+
+        Parameters
+        ----------
+        master_edge : int
+            Master edge id.
+        slave_edge : int
+            Slave edge id.
+        reverse_master : bool, optional
+            Set to ``True`` to reverse V direction.
+        reverse_slave : bool, optional
+            Set to ``True`` to reverse U direction.
+        same_as_master : bool, optional
+            Set to ``False`` to reverse Master/Slave Relation.
```

Comment:
```suggestion
            Whether the directions of the slave edge and master edge are the same. The default is ``True``. 
            Set to ``False`` to reverse the master and slave relation.
```

Reply:
I'm not sure I fully understand this option based on what was initially there.

Reply:
@maxcapodi78 can you clear up exactly what `same_as_master` is?

Reply:
same_as_master has nothing with directions. it means B Field of Master = B Field of Slave (in case of True) B Field of Master = -BField of Slave (in case of False)

Reply:
@jleonatti Do you understand Massimo's description? If so, can you edit it in the file?

Reply:
@PipKat Massimo changed it to:

"Whether the B-Field of the slave edge and master edge are the same. The default is ``True``." I think this description works. Is it okay for you?

Reply:
@jleonatti Absolutely! This makes sense--though I am wondering if B-Field is an actual field name that should be tagged as a literal string.